drift-transport-netty/src/test/java/com/facebook/drift/transport/netty/codec/TestHeaderTransport.java

```diff
@@ -98,6 +170,8 @@ private static ByteBuf createTestMessage(ByteBufAllocator allocator, String meth
             ProtocolWriter writer = new ProtocolWriter(protocolWriter);
             writer.writeStructBegin("method_args");
             writer.writeStructEnd();
+            writer.writeStructBegin(Strings.repeat("a very long string", 1000));
```

Comment:
This doesn't increase the message size, as it doesn't store the struct name in the actual Thrift output. This is quite confusing =\

Additionally `Strings.repeat("a very long string", 1000)` will be perfectly compressed to ~20 bytes, and that's not the case we are trying to excersize.

What we actually want to do, is to create a message that is, say 5kb long. Then create a slice of, say 100bytes and verify that the `FrameInfo` is succesfully decoded.

Instead of `writer.writeStructBegin(Strings.repeat("a very long string", 1000));` you can do `writer.writeString();` in between of `writer.writeStructBegin("method_args");` and ` writer.writeStructEnd();`. Make sure that the string is not compressable.

Reply:
Thanks, yes, checking the BinaryProtocol, it seems it is basic ignore the name here.  I will read a large randomly generated string instead (have to save it in a txt file though).